### DISCUSSION TITLE
Conditional -morpho

### DISCUSSION THREAD

**Discussion by dikshanshs** [neperfepx/neper#413] [Q&A] [answered]
@rquey sir,
                 Can we use the if-else condition in morpho
for example: 
     if grain_size>some value then lamellar else normal grain growth.

**Comment by rquey** [comment, marked as answer]
For now, if-else conditions cannot be used for expressions, so, it takes 2 steps:

```
neper -T -n 10 -statcell diameq
awk '{if ($1>0.6) {print "lamellar(w=0.1)"} else {print "gg(0.3)"}}' n10-id1.stcell | nl > morpho_msfile
neper -T -n 10::from_morpho -morpho "voronoi::msfile(morpho_msfile)" -o d
```

Visualize:
```
neper -V d.tess -datacellcol "scaleid(1)" -datacelltrs 0.5 -print img
```
![image](https://user-images.githubusercontent.com/2659376/159437557-914a550a-0b7d-4e9a-ab05-f8b763182e2c.png)
